src/pages/LandingPages/MGCovid19/MGCovidHero.vue

```diff
@@ -95,6 +95,9 @@ export default {
 		category: {
 			type: String,
 			default: 'covid-relief'
+		},
+		isExperimentActive: {
```

Comment:
It doesn't look like there are any usages of this currently. What is this going to be used for? Is that code still in progress?

Reply:
@emuvente good catch. will remove it. thanks